ui/components/app/assets/util/buildSymbolGroup.ts

```diff
@@ -0,0 +1,10 @@
+export const buildSymbolGroup = (symbols: string[]): string => {
+  if (symbols.length === 1) {
+    return `${symbols[0]} only`;
+  } else if (symbols.length === 2) {
+    return `${symbols[0]} +${symbols.length - 1} other`;
+  } else if (symbols.length > 2) {
+    return `${symbols[0]} +${symbols.length - 1} others`;
+  }
+  return '';
+};
```

Comment:
nit - since we are returning in each block, we don't need to else-if/else statements - can just use separate if statements.

This is a nice separate function though. [In mobile](https://github.com/MetaMask/metamask-mobile/pull/13925#discussion_r2081419734) we used an intermediate obj to avoid nested ternaries 😄